resource-definitions/volume-pvc/volumes/README.md

```diff
@@ -0,0 +1 @@
+This example illustrates how a Workload can use persistent storage service through the Kubernetes [volumes](https://kubernetes.io/docs/concepts/storage/volumes) system. It uses the [`volume-pvc`](https://developer.humanitec.com/integration-and-extensions/drivers/volume-driver/persistent-volume-claim/) Driver to create a [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for your application. If you have special requirements for your PersistentVolume, you can also use the [Template Driver](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/template/) to create it as shown in [this other example](../../template-driver/volumes-static-provisioning/).
```

Comment:
Nit: relative links will not work in the rendered site on developer.humanitec.com/examples . I suggest to use the absolute link to the examples site instead.

```suggestion
This example illustrates how a Workload can use persistent storage service through the Kubernetes [volumes](https://kubernetes.io/docs/concepts/storage/volumes) system. It uses the [`volume-pvc`](https://developer.humanitec.com/integration-and-extensions/drivers/volume-driver/persistent-volume-claim/) Driver to create a [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for your application. If you have special requirements for your PersistentVolume, you can also use the [Template Driver](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/template/) to create it as shown in [this other example](https://developer.humanitec.com/examples/resource-definitions/template-driver/volumes-static-provisioning/).
```